sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_v1/query.sql

```diff
@@ -52,6 +52,19 @@ fenix_all_events AS (
   FROM
     `moz-fx-data-shared-prod.org_mozilla_firefox.events`
 ),
+ios_all_events AS (
+  SELECT
+    submission_timestamp,
+    events
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_ios_firefox.events`
+  UNION ALL
+  SELECT
+    submission_timestamp,
+    events
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_ios_firefoxbeta.events`
```

Comment:
Should we also be including `org_mozilla_ios_fennec`?

Reply:
Yeah, I guess we should